packages/browser/src/index.test.ts

```diff
@@ -39,17 +39,19 @@ const requester = jest.fn();
 const failingRequester = jest.fn().mockRejectedValue(new Error('Failed!'));
 const currentUnixTimeStamp = Date.now() / 1000;
 
+const fetchOidcConfig = jest.fn(async () => ({
+  authorizationEndpoint,
+  userinfoEndpoint,
+  tokenEndpoint,
+  endSessionEndpoint,
+  revocationEndpoint,
+  jwksUri,
+  issuer,
+}));
+
 jest.mock('@logto/js', () => ({
   ...jest.requireActual('@logto/js'),
-  fetchOidcConfig: jest.fn(async () => ({
-    authorizationEndpoint,
-    userinfoEndpoint,
-    tokenEndpoint,
-    endSessionEndpoint,
-    revocationEndpoint,
-    jwksUri,
-    issuer,
-  })),
+  fetchOidcConfig: async () => fetchOidcConfig(),
```

Comment:
should use `setTimeout` to simulate network?

Reply:
Updated. PTAL.